res/css/views/context_menus/_IconizedContextMenu.scss

```diff
@@ -50,21 +50,21 @@ limitations under the License.
         }
```

Comment:
> You should probably also add this menu in the room info panel or similar, since unlike in your screenshot the closest date header might be pages and pages back if the room is somewhat active.
>
> https://github.com/matrix-org/matrix-react-sdk/pull/7317#issuecomment-991163942

As separate iterations, the plan is to make the date headers sticky.

And add a slash command `/jumptodate 2021-12-10`